src/blog/2025-09-19-oss-summit-europe-2025.md

```diff
@@ -0,0 +1,101 @@
+* **Instrument the handoffs.** Track how DevRel content and support reduce common blockers in proofs of concept. Track time from the first question to unblock. Track how often a successful proof of concept moves to a trial or purchase.  
+* **Capture stories and systematize them.** When you help a team move from evaluation to purchase faster, write the play. Which persona? Which blockers? Which content or support solved it? Turn that into a repeatable motion that anyone in your company can use.
+
+The Metrics working group is rebooting to publish examples you can adapt. Expect templates for funnel definitions, example dashboards, and sample reports that map DevRel outcomes to pipeline stages and customer health.
+
+### What are the most common challenges?
+
+We heard three over and over again.
+
+* **Perception problem.** DevRel is considered a cost center without clear value. The remedy is consistent language, transparent goals, and outcome reports that map to company OKRs.  
+* **Silos and access.** Teams do not have access to product analytics or cannot connect community signals to product data. The remedy is cross-functional alignment and a minimum shared instrumentation plan. Start small. Agree on the few events and labels everyone will use.  
+* **Unsustainable motions.** Travel-heavy schedules burn people out and waste money. Replace a portion of travel with scalable digital programs, local community partners, and content that compounds over time. When you do travel, tie every trip to a measurable goal.
+
+The Foundation will publish playbooks to facilitate cross-functional collaboration and promote sustainable operating models. These will include checklists, templates for shared dashboards, and example rhythms for data reviews involving product, marketing, and sales teams.
+
+### Which skills matter most now?
+
+This is where the craft of DevRel meets the discipline of product.
+
+* **Authentic, developer-first communication.** Speak to the problems developers actually have. Show how to solve them with real code, real examples, and real constraints.  
+* **Empathy and persona awareness.** Know who you are helping. A senior platform engineer has different constraints than a startup founder. Route your work accordingly.  
+* **Technical advocacy or strong routing to technical owners.** You do not need to be the deepest expert in every stack. You do need to get developers unblocked quickly. Either do it yourself or build the internal routing that makes it happen.  
+* **Data literacy.** You do not need to be a data scientist. You do need to choose the right metrics, interpret what they mean, and tell a clear business story. Learn enough SQL or analytics tooling to answer basic questions yourself. Pair with data partners for the rest.
+
+These skills are learnable. The Foundation will gather examples and training paths you can use to level up your team.
```

Comment:
These skills are learnable. The [Resources Working Group](https://github.com/DevRel-Foundation/wg-resource-aggregation) will gather examples and training paths into a [Career Manual](https://github.com/DevRel-Foundation/wg-resource-aggregation) you can use to level up your team.